sdk.nix

```diff
@@ -40,7 +40,7 @@ rec {
     packageOverrides = pkgs: rec {
       inherit (ugpkgs) docker go imagemagick linux nix;
       mysql = ugpkgs.mariadb;
-      php = ugpkgs.php54;
+      php = ugpkgs.php53;
```

Comment:
i think that's redundant as we're going to override php only for one venture seafas